Mathlib/RepresentationTheory/GroupCohomology/LowDegree.lean

```diff
@@ -223,6 +223,12 @@ def twoCocycles : Submodule k (G × G → A) := LinearMap.ker (dTwo A)
 
 variable {A}
 
+instance : FunLike (oneCocycles A) G A := ⟨(·.val), Subtype.val_injective⟩
```

Comment:
Can you update the code below (e.g. `f.1 (g * h)` in `oneCocycles_map_mul_of_isTrivial`) to use this instance?

Reply:
Thank you for the review. They are updated in the commit 2d277c58c4cb6e88baf5beb34ec9f666047cfdbf.